aws/templates/solution/solution_ecs.ftl

```diff
@@ -198,14 +240,25 @@
                         "ecs": {
                             "commands":
                                 attributeIfTrue(
-                                    "01Fluentd"
+                                    "01Fluentd",
                                     defaultLogDriver == "fluentd",
                                     {
                                         "command" : "/opt/codeontap/bootstrap/fluentd.sh",
                                         "ignoreErrors" : "false"
                                     }) +
+                                attributeIfTrue(
+                                    "02ConfigureEFSClusterWide",
+                                    ecs.ClusterWideStorage?has_content && ecs.ClusterWideStorage == true,
```

Comment:
don't check has_content - it always will

Reply:
Fixed